### ISSUE TITLE
Command.StandardOutput.ReadLine() is unreadable

### ISSUE THREAD

**Issue by xjfnet** [madelson/MedallionShell#10]
```csharp
var line = cmd.StandardOuput.ReadLine();
```
line is unreadable : "?????????????"

```csharp
using(var sr = new StreamReader(cmd.StandardOutput.BaseStream, Encoding.Default))
{
    var line = sr.ReadLine();
}
```
line is readable : "some unicode char"

**Comment by xjfnet** [comment]
just a little change to the source code, it will work well
```csharp
        public InternalProcessStreamReader(StreamReader processStreamReader)
        {
            this.processStream = processStreamReader.BaseStream;
            this.pipe = new Pipe();
            /* set encoding as the origin StreamReader */
            this.reader = new StreamReader(this.pipe.OutputStream, processStreamReader.CurrentEncoding);
            this.task = Task.Run(() => this.BufferLoop());
        }

```

**Comment by madelson** [comment]
Will be addressed by https://github.com/madelson/MedallionShell/pull/11

**Comment by madelson** [comment]
Fixed as of 1.3.0 (https://www.nuget.org/packages/MedallionShell/1.3.0)
